digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala

```diff
@@ -288,6 +293,50 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
     RoadLinkDAO.LinkTypeDao.getAllLinkType(linkIds).groupBy(_._1)
   }
 
+  def getAllPrivateRoadAssociationNames(newTransaction: Boolean = true): Seq[String] = {
+    if(newTransaction) {
+      withDynSession {
+        LinkAttributesDao.getAllExistingDistinctValues(privateRoadAssociationPublicId)
+      }
+    } else
+      LinkAttributesDao.getAllExistingDistinctValues(privateRoadAssociationPublicId)
+  }
```

Comment:
these transaction-conditions are true by default, can be removed, that method is only use on digiroadApi